### PR TITLE
fix/Container estimations contract call

### PR DESCRIPTION
**Pull request by carpawell**
Relates #1506. Does not fix (of does fix, not sure yet but that is the first time I [can see](https://github.com/nspcc-dev/neofs-node/issues/1506#issuecomment-1418845811) at least one metric in the contract locally).

### CODE REVIEW

File: pkg/morph/client/container/client.go [hunk 1]
```diff
@@ -89,9 +89,15 @@ type opts struct {
 }
 
 func defaultOpts() *opts {
-	var o = new(opts)
-	o.staticOpts = append(o.staticOpts, client.TryNotary())
-	return o
+	return new(opts)
```
Comment (roman-khimov) [inline pkg/morph/client/container/client.go:92]: Seems too dangerous to me. Can we have `DisableNotary()` specifically to use in those cases where its' needed?

Comment (roman-khimov) [reply pkg/morph/client/container/client.go:92]: And to be fair it's not about enabling/disabling notary, it's more about using appropriate signers. This `TryNotary()` name always looked wrong to me.

File: pkg/morph/client/container/client.go [hunk 2]
```diff
@@ -85,13 +89,24 @@ type opts struct {
...
+}
+
+// DisableNotarySinging returns option to disable
+// notary request signing. With that option, every
+// call will be created, signed with provided key
+// (a single regular sign) and sent to the side
+// chain.
+func DisableNotarySinging() Option {
```
Comment (roman-khimov) [inline pkg/morph/client/container/client.go:106]: Some notary can sing probably, but they more often sign.

File: pkg/morph/client/container/client.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/morph/client/container/client.go:98]: So we still change default?

File: pkg/morph/client/container/client.go [hunk 3]
```diff
@@ -85,13 +89,24 @@ type opts struct {
...
+	return func(o *opts) {
+		o.staticOpts = append(o.staticOpts, client.TryNotary())
```
Comment (roman-khimov) [inline pkg/morph/client/container/client.go:108]: Is it "disable" or "enable"?

Comment (carpawell) [reply pkg/morph/client/container/client.go:108]: wow, hasn't committed this line changed

File: pkg/morph/client/container/client.go (hunk 1 above)
Comment (carpawell) [reply pkg/morph/client/container/client.go:98]: i pushed smth strange, check one more time, please

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**